docs/customizations/footer.md

```diff
@@ -0,0 +1,218 @@
+# Footer
+
+## Table of contents
+
+- [Interfaces](#interfaces)
+  - [IFooterProps](#ifooterprops)
+  - [IFooterComponentOverrides](#ifootercomponentoverrides)
+  - [IFooterControlProps](#ifootercontrolprops)
+  - [IFooterStyleProps](#ifooterstyleprops)
+- [Sample Scenarios](#sample-scenarios)
+  - [Replacing default sub-components with custom components](#replacing-default-sub-components-with-custom-components)
+  - [Using a custom send box in the footer](#using-a-custom-send-box-in-the-footer)
+  - [Overriding default button behaviors](#overriding-default-button-behaviors)
+
+## Interfaces
+
+### [IFooterProps](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-components/src/components/footer/interfaces/IFooterProps.ts)
+
+The top-level interface for customizing `Footer`.
+
+| Attribute | Type | Required | Description | Default |
+| - | - | - | - | - |
+| componentOverrides     | [IFooterComponentOverrides](#ifootercomponentoverrides)     | No | Used for overriding default `Footer` components, including all the default buttons | -
+controlProps | [IFooterControlProps](#ifootercontrolprops) | No | Properties that control the element behariors | -
+styleProps | [IFooterStyleProps](#ifooterstyleprops) | No | Properties that control the element styles | -
+
+### [IFooterComponentOverrides](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-components/src/components/footer/interfaces/IFooterComponentOverrides.ts)
+
+Custom React components can be passed as input to override the default sub-components. Alternatively, you can stringify the React component before passing it in. The `chat-components` library provides one util function that can be used: [encodeComponentString](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-components/src/common/encodeComponentString.ts).
+
+| Attribute | Type | Required | Description | Default | 
+| - | - | - | - | - |
+| DownloadTranscriptButton     | ReactNode\|string     | No | Used for overriding default download transcript button | -
+EmailTranscriptButton | ReactNode\|string | No | Used for overriding default email transcript button | -
+AudioNotificationButton | ReactNode\|string | No | Used for overriding default audio toggle button | -
+
+### [IFooterControlProps](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-components/src/components/footer/interfaces/IFooterControlProps.ts)
+
+| Attribute | Type | Required | Description | Default | 
+| - | - | - | - | - |
+| id     | string     | No | The top-level element id for the footer | "oc-lcw-footer"
```

Comment:
default seems to be : lcw-components-footer 

https://github.com/microsoft/omnichannel-chat-widget/blob/75135c54d4377cc372876e0a57743df3e86d2bd3/chat-components/src/common/Constants.ts#L86